pom.xml

```diff
@@ -907,6 +922,10 @@
                     <artifactId>java17</artifactId>
                     <version>1.0</version>
                   </signature>
+                  <ignores>
+                    <ignore>org.apache.hadoop.conf.Configuration</ignore>
+                    <ignore>org.apache.hadoop.fs.FileSystem</ignore>
```

Comment:
Does this mean Hadoop3 now requires Java 1.8?

Reply:
FWIW, I would not have a problem making 1.9.0 also require Java 8.

Reply:
I haven't explicitly checked, but I wouldn't know why Hadoop wouldn't force this move given Java7 being EOL'ed forever and a half by now.

Reply:
I agree. I don't know why they wouldn't either, and it looks like they did: I found `<javac.version>1.8</javac.version>` in `hadoop-project/pom.xml`

Accumulo was reluctant before, without a major version number bump. I wonder if there'd be less resistance now. :man_shrugging: